runtime/oti/j9nonbuilder.h

```diff
@@ -4359,6 +4359,8 @@ typedef struct J9MemoryManagerFunctions {
 	j9objectmonitor_t*  ( *j9gc_objaccess_getLockwordAddress)(struct J9VMThread *vmThread, j9object_t object) ;
 	void  ( *j9gc_objaccess_cloneObject)(struct J9VMThread *vmThread, j9object_t srcObject, j9object_t destObject) ;
 	void ( *j9gc_objaccess_copyObjectFields)(struct J9VMThread *vmThread, J9Class* valueClass, j9object_t srcObject, UDATA srcOffset, j9object_t destObject, UDATA destOffset) ;
+	void ( *j9gc_objaccess_copyObjectFieldsToArrayElement)(struct J9VMThread *vmThread, J9Class *arrayClazz, j9object_t srcObject, J9IndexableObject *arrayRef, U_32 index) ;
+	void ( *j9gc_objaccess_copyObjectFieldsFromArrayElement)(struct J9VMThread *vmThread, J9Class *arrayClazz, j9object_t destObject, J9IndexableObject *arrayRef, U_32 index) ;
```

Comment:
not sure what's right, but index in some other APIs was I_32

Reply:
at various other places in the code I saw I_32, U_32, UDATA.... Again, not sure what' right, but at least let's be consistent in this file) and use I_32